docs/docs/maintaining-a-plugin.md

```diff
@@ -1,10 +1,61 @@
 ---
 title: Maintaining a Plugin
-issue: https://github.com/gatsbyjs/gatsby/issues/21067
 ---
 
-This is a stub article meant to be filled with tips on how to maintain a Gatsby plugin once you've published it as an npm package.
+The Gatsby community thrives on the power of plugins. You often find that there is a plugin for almost everything ranging from source plugins to transformer plugins. A useful way of understanding the purpose of a plugin at a glance is by its name. Check out the [guide to naming conventions for plugins](/docs/naming-a-plugin/) to learn more.
 
-Topics to be covered:
+> Wondering how to [create a plugin](/docs/creating-plugins)? Look no further. Start contributing.
 
-- yarn workspaces can solve yarn link inconsistencies
+## What is a plugin?
+
+Gatsby plugins are Node.js packages that implement Gatsby APIs. To learn more take a look at the [plugin documentation](/docs/plugins/).
+
+In creating plugins, the bulk of the work is during the development phase. However, there’s still a need to take a look at the dependencies and security features from time to time. Keeping Plugins up to date becomes really tasking and it is important to be careful when updating dependencies seeing as this could potentially break your apps.
```

Comment:
```suggestion
In creating plugins, the bulk of the work is during the development phase. However, there’s still a need to take a look at the dependencies and security features from time to time. Keeping plugins up to date can become really tasking and it is important to be careful when updating dependencies, seeing as this could potentially break users' apps.
```
It would be good to focus on the end user of the plugins to motivate why they'd want to update.